srcpkgs/conduit/template

```diff
@@ -0,0 +1,33 @@
+# Template file for 'conduit'
+pkgname=conduit
+version=0.3.0
+revision=1
+wrksrc="conduit-v${version}"
+build_style=cargo
+build_helper=rust
+hostmakedepends="clang"
+short_desc="Simple, fast and reliable chat server powered by Matrix"
+maintainer="Joel Beckmeyer <joel@beckmeyer.us>"
+license="Apache-2.0"
+homepage="https://conduit.rs/"
+distfiles="https://gitlab.com/famedly/conduit/-/archive/v${version}/conduit-v${version}.tar.bz2"
+checksum=1a3281542f578764ee7a3e48138bcd5d9df20f5350c8d3fbd0f5053c493f0028
+conf_files="/etc/conduit/conduit.toml"
+
+system_accounts="_conduit"
+_conduit_homedir="/var/lib/conduit"
+
+make_dirs="/var/lib/conduit 0700 _conduit _conduit"
+
+export BINDGEN_EXTRA_CLANG_ARGS="-I${XBPS_CROSS_BASE}/usr/include"
+if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
+	makedepends+=" libatomic-devel"
+	XBPS_CROSS_RUSTFLAGS+=" -latomic"
+fi
+
+post_install() {
+	vmkdir etc/conduit
+	vcopy conduit-example.toml etc/conduit/conduit.toml
```

Comment:
Sample config files should be installed using `vsconf`. That would put it into `/usr/share/examples/conduit`, which would be more appropriate anyway, IMO, as it makes it less probable that someone just installs conduit and enables it with the example config without adjusting stuff like the server domain.